libs/frontend/feature-build/src/components/build-render/ProseMirrorEditor.vue

```diff
@@ -53,7 +53,7 @@ const mountProseMirrorEditor = (
     editor.editView?.destroy()
     editor.editView = createComponentEditorView(
       {
-        content: selectedComponent.content ?? '',
+        content: selectedComponent.content || '<div class="pm-p"></div>',
```

Comment:
I did this to try making empty prosemirror components selectable, but it doesn't work. I'll add an issue later for improving this.